src/services/codeFixProvider.ts

```diff
@@ -124,9 +124,15 @@ export function eachDiagnostic(context: CodeFixAllContext, errorCodes: readonly
 }
 
 function getDiagnostics({ program, sourceFile, cancellationToken }: CodeFixContextBase) {
-    return [
+    const diagnostics = [

```

Comment:
Yeah I'm not sure why I don't see codefixes. Adding some console logs, I'm surprised to see that this code path isn't reached when running with ts-fix.

Reply:
It looks like ts-fix does this: https://github.com/microsoft/ts-fix/blob/main/src/index.ts#L341

So it's not using this path at all.

Reply:
To follow-up, I pulled this branch and rebuilt from scratch as well as reinstalled ts-fix from scratch. I can see the quick fix in VSCode now, but ts-fix still doesn't report any diagnostics to fix.

Reply:
Yeah, https://github.com/microsoft/ts-fix/blob/2898fac87de321fe5d455b594f874bbcafbab6bd/src/index.ts#L343 needs to be updated to also call `getDeclarationDiagnostics`. You should be able to have it do `[...project.program.getSemanticDiagnostics(file), ...project.program.getDeclarationDiagnostics(file)`.

Reply:
Aha, I found the issue. ts-fix is only calling `getSemanticDiagnostics` but isolatedDeclarations are from `getDeclarationDiagnostics`. I don't know if this is considered working as intended, but I submitted a pull request nonetheless https://github.com/microsoft/ts-fix/pull/30.

Reply:
@jakebailey ah I didn't see your comment (still had this window open while investigating and didn't refresh)

Reply:
Resolving this thread; we can talk about ts-fix on the other repo since that doesn't block this PR.